src/js/disability-benefits/526EZ/components/ReviewCardField.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+
+import {
+  getDefaultFormState,
+  getDefaultRegistry
+} from '@department-of-veterans-affairs/react-jsonschema-form/lib/utils';
+
+import { errorSchemaIsValid } from '../../../common/schemaform/validation';
+
+import set from '../../../../platform/utilities/data/set';
+import get from '../../../../platform/utilities/data/get';
+
+
+/**
+ * Displays a review card if the information inside is valid.
+ *
+ * For use on a schema of type 'object' or 'array'.
+ * Intended to wrap objects or arrays to avoid duplicate functionality here.
+ */
+export default class ReviewCardField extends React.Component {
+  static defaultProps = {
+    uiSchema: {},
+    errorSchema: {},
+    idSchema: {},
+    registry: getDefaultRegistry(),
+    required: false,
+    disabled: false,
+    readonly: false,
+  }
+
+
+  constructor(props) {
+    super(props);
+
+    // Throw an error if there’s no viewComponent (should be React component)
+    if (typeof get('ui:options.viewComponent', this.props.uiSchema) !== 'function') {
+      throw new Error(`No viewComponent found in uiSchema for ReviewCardField ${this.props.idSchema.$id}.`);
+    }
+
+    if (!['object', 'array'].includes(this.props.schema.type)) {
+      throw new Error(`Unknown schema type in ReviewCardField: ${this.props.schema.type}`);
+    }
+
+    this.state = {
+      // Set initial state based on whether all the data is valid
+      editing: !errorSchemaIsValid(props.errorSchema)
+    };
+  }
+
+  onPropertyChange(name) {
```

Comment:
Looks like this is the same change handler as `ObjectField` - de-duplicating these might be a bad abstraction, but thought I'd at least point out that this handler already lives somewhere else. Thoughts?

Reply:
Yeah, I just copied this straight over from `ObjectField`. Good catch. I don't think it makes sense to abstract this out. We'd then put ourselves in potentially a weird spot where two unrelated fields share the same helper functions. If we then have to update it for one...

I dunno. It seems small enough I wasn't going to worry about it, to be honest.